app/views/layouts/application.html.erb

```diff
@@ -14,8 +14,9 @@
       </button>
       <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
         <div class="navbar-nav">
-          <a class="nav-item nav-link active" href="/dashboard">Dashboard<span class="sr-only">(current)</span></a>
-          <a class="nav-item nav-link" href="/gardens">My Gardens</a>
+          <%= link_to 'Dashboard', dashboard_path, class: "nav-item nav-link"%>
+          <%= link_to 'My Gardens', gardens_path, class: "nav-item nav-link"%>
+          <%= link_to 'Schedule', schedules_path, class: "nav-item nav-link"%>
```

Comment:
calling this Watering Schedule might provide more clarity, but I'm glad it's in the nav bar!